### PR TITLE
Added user access section

### PR DESCRIPTION
**Pull request by shrinathprabhu**
Resolves [AR-2255](https://team-1624093970686.atlassian.net/browse/AR-2255) and [AR-2256](https://team-1624093970686.atlassian.net/browse/AR-2256).

## Changes
- Added static UserAccess section
- Added ChainType Selection section with radio button group and appropriate store operations

## Screenshots
Screenshot from 2022-05-06 07-57-34

## Checklist

- [x] The branch name follows the format: `developer/AR-XXX-issue-name`.
- [x] The changes have been tested locally.

### CODE REVIEW

File: src/store/configure.store.ts [hunk 1]
```diff
@@ -1,19 +1,29 @@
...
   appName: string
+  access: {
+    selectedChain: string
+  }
 }
 
 const state: ConfigureState = {
   appName: '',
+  access: {
+    selectedChain: '',
```
Comment (ajithranka) [inline src/store/configure.store.ts:11]: Should we set the default value to Ethereum?

File: src/components/app-configure/access/ChainType.vue [hunk 2]
```diff
@@ -0,0 +1,53 @@
+<script lang="ts" setup>
+import { computed, type ComputedRef } from 'vue'
+import { useStore } from 'vuex'
+
+import SettingCard from '@/components/app-configure/SettingCard.vue'
+import VCard from '@/components/lib/VCard/VCard.vue'
+import VRadioGroup from '@/components/lib/VRadioGroup/VRadioGroup.vue'
+import VStack from '@/components/lib/VStack/VStack.vue'
+import { chains } from '@/utils/constants'
```
Comment (ajithranka) [inline src/components/app-configure/access/ChainType.vue:9]: In the constants file, add a chain and option type and use it to type the options.

```
type Chain = "ethereum" | "polygon" | "binance"

type Option<T> = { label: string, value: T }

const CHAIN_OPTIONS: Option<Chain>[] = [
  { label: "Ethereum", value: "ethereum" },
  { label: "Polygon", value: "polygon" },
  { label: "Binance", value: "binance" },
]
```

File: src/components/app-configure/access/ChainType.vue [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+const store = useStore()
+const selectedChain: ComputedRef<string> = computed(
+  () => store.getters.selectedChain
+)
+
+function handleChangeChainType(value: string) {
```
Comment (ajithranka) [inline src/components/app-configure/access/ChainType.vue:16]:
```suggestion
function handleChangeChainType(value: Chain) {
```

File: src/store/configure.store.ts [hunk 4]
```diff
@@ -1,19 +1,29 @@
 type ConfigureState = {
   appName: string
+  access: {
+    selectedChain: string
```
Comment (ajithranka) [inline src/store/configure.store.ts:4]:
```suggestion
    selectedChain: Chain
```

File: src/store/configure.store.ts [hunk 5]
```diff
@@ -1,19 +1,29 @@
...
 const getters = {
   appName: (state: ConfigureState) => state.appName,
+  selectedChain: (state: ConfigureState) => state.access.selectedChain,
 }
 
 const mutations = {
   updateAppName(state: ConfigureState, appName: string) {
     state.appName = appName
   },
+  updateSelectedChain(state: ConfigureState, selectedChain: string) {
```
Comment (ajithranka) [inline src/store/configure.store.ts:24]:
```suggestion
  updateSelectedChain(state: ConfigureState, selectedChain: Chain) {
```

File: src/store/configure.store.ts (hunk 1 above)
Comment (karthik-durai) [reply src/store/configure.store.ts:11]: Yeah, I guess any one should be default

**Review by karthik-durai — APPROVED** [review]

**Pull request merged**